terraform.tfvars

```
@@ -110,6 +110,11 @@ projects = [
     display_name  = "Send Letter Service"
     azuread_group = "dcd_group_bulkscan_v2"
   },
+  {
+    name          = "bulk-scan-processor"
+    display_name  = "Bulk Scan Processor"
+    azuread_group = "dcd_group_bulkscan_v2"
```

Generally you should have a team project not a microservice one.

I'd suggest this:
```suggestion
    name          = "bulk-scan"
    display_name  = "Bulk Scan"
    azuread_group = "dcd_group_bulkscan_v2"
```